db_updates/updates.sql

```diff
@@ -0,0 +1,85 @@
+DROP TABLE IF EXISTS articles_updated CASCADE;
+DROP TABLE IF EXISTS experiments_updated CASCADE ;
+DROP TABLE IF EXISTS locations_updated CASCADE ;
+DROP TABLE IF EXISTS tags_updated CASCADE;
+DROP TABLE IF EXISTS votes_updated CASCADE;
+
+CREATE TABLE articles_updated(
+  uniqueid bigserial PRIMARY KEY,
+  "timestamp" TIMESTAMP,
+   authors text,
+   title text,
+   abstract text,
+   reference text,
+   pmid VARCHAR(64),
+   doi VARCHAR(128),
+   neurosynthid VARCHAR(64)
+);
+
+CREATE TABLE experiments_updated(
+  experiment_id bigserial PRIMARY KEY,
+  title text,
+  caption text,
+  flagged BOOLEAN,
```

Comment:
Will discuss whether we want to i) allow no user modifications to Brainspell's DB, ii) allow modifications, but only by pushing a GitHub clone of the article, iii) allow modifications as usual (but would likely put this as last priority, since it's not relevant to meta-analyses)